addenda15_test.go

```diff
@@ -70,14 +70,10 @@ func BenchmarkAddenda15Parse(b *testing.B) {
 func testAddenda15ValidRecordType(t testing.TB) {
 	addenda15 := mockAddenda15()
 	addenda15.recordType = "63"
-	if err := addenda15.Validate(); err != nil {
-		if e, ok := err.(*FieldError); ok {
-			if e.FieldName != "recordType" {
-				t.Errorf("%T: %s", err, err)
-			}
-		} else {
-			t.Errorf("%T: %s", err, err)
-		}
+	err := addenda15.Validate()
+	// TODO: are we not expecting any errors here?
```

Comment:
return the fieldError

	if addenda15.recordType != "7" {
		fieldError("recordType", NewErrRecordType(7), addenda15.recordType)
	}